src/Explorer/QueryCopilot/QueryCopilotTab.tsx

```diff
@@ -56,7 +57,7 @@ interface SuggestedPrompt {
 }
 
 interface QueryCopilotTabProps {
-  initialInput: string;
+  queryCopilotState: QueryCopilotState;
```

Comment:
I don't think you should pass the states via props. You can simply add a hook to import the states in this file.